drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -97,74 +97,53 @@ LOG_MODULE_REGISTER(espi, CONFIG_ESPI_LOG_LEVEL);
  *  46h   | SMVW05 | generic      | generic      | generic   | generic     |
  *  47h   | MSVW07 | res          | res          | res       | HOST_C10    |
  *  4Ah   | MSVW08 | res          | res          | DNX_WARN  | res         |
+ * These are configurable by overriding device tree vw routing             |
+ *  50h   | SMVW06 | gpio_3       | gpio_2       | gpio_1    | gpio_0      |
+ *  51h   | SMVW07 | gpio_7       | gpio_6       | gpio_5    | gpio_4      |
+ *  52h   | SMVW08 | ocb_3        | ocb_2        | ocb_1     | ocb_0       |
```

Comment:
@jvasanth1 this is index mapping is not correct 
![image](https://user-images.githubusercontent.com/16172078/219719410-ee8054ad-ae57-44dc-85b5-467bab84f8b1.png)